ops/docker/op-stack-go/Dockerfile

```diff
@@ -233,6 +233,19 @@ ADD "https://github.com/EspressoSystems/ark-srs/releases/download/v0.2.0/kzg10-a
 COPY --from=op-batcher-builder /app/op-batcher/bin/op-batcher /usr/local/bin/
 CMD ["op-batcher"]
 
+FROM $TARGET_BASE_IMAGE AS op-batcher-enclave-target
+ARG ENCLAVE_BATCHER_ARGS=""
+# CGO dependencies
+RUN apk add gcc
+# enclave-entrypoint dependencies
+RUN apk add socat bash trurl curl
+ENV AZTEC_SRS_PATH /aztec/kzg10-aztec20-srs-1048584.bin
```

Comment:
Why do we need this?

Reply:
This is needed by espresso-network-go library, one of the rust crypto dependencies needs some precomputed data. It tries to download it on its own otherwise, which might not work inside the enclave.